docs/content/drafts/MarkdownEditor.mdx

```diff
@@ -28,9 +28,8 @@ import {MarkdownEditor} from '@primer/react/drafts'
 
 A `Label` is always required for accessibility:
 
-{/* prettier-ignore-start */}
```

Comment:
@josepmartins I'm not seeing the these mdx changes I made here in the [github-pages deployment](https://primer-cf028944c7-13348165.drafts.github.io/drafts/MarkdownEditor). Do you know why the changes would not be reflected? 

Also, let me know if there is any additional steps to move this PR forward.